autoload/jedi.vim

```diff
@@ -222,6 +222,16 @@ function! jedi#force_py_version_switch()
 endfunction
 
 
+function! jedi#auto_put_import()
+    " Auto put import statement after from module.name<space> and complete
+    if search('\<from\s\+[A-Za-z0-9._]\+\s*\%#\s*$', 'bcn', line('.'))
```

Comment:
The `\<` is a word boundary, but not necessarily whitespace, yes.

I would also anchor it at the beginning: `^\s\+